releasetasks/release_configs/dev_mozilla-release_fennec_rc_graph_2.yml

```diff
@@ -23,7 +23,8 @@ update_verify_enabled: false
 push_to_releases_enabled: true
 # rc graph 2 only needs final_verify_channels
 channels:
-publish_to_balrog_channels: []
+publish_to_balrog_channels:
+    - "release-dev"
```

Comment:
@rail shouldn't we use here `release`? The configs are switching gears to use the staging Balrog instance anyway. I couldn't find any `release-dev` channel there. I'm not sure how this works in Firefox 😕